packages/asset/hardhat.config.ts

```diff
@@ -0,0 +1,54 @@
+import { HardhatUserConfig } from "hardhat/config";
+
+import "@nomicfoundation/hardhat-toolbox";
+import "@nomiclabs/hardhat-ethers";
+import "hardhat-deploy";
+
+const config: HardhatUserConfig = {
+  solidity: {
+    compilers: [
+      {
+        version: "0.8.18",
+        settings: {
+          optimizer: {
+            enabled: true,
+            runs: 2000,
+          },
+        },
+      },
+    ],
+  },
+  namedAccounts: {
+    deployer: {
+      default: 0,
+    },
+    upgradeAdmin: {
+      default: 1,
```

Comment:
Will need updating to use sandAdmin and to specify the addresses on mainnet / polygon